uploadNeuroDB/mass_nii.pl

```diff
@@ -1,5 +1,33 @@
 #!/usr/bin/perl
 
+=pod
+
+=head1 NAME
+
+mass_nii.pl -- Generates NIfTI files based on the inserted MINC files and
+insert them into the LORIS database.
+
+=head1 SYNOPSIS
+
+perl mass_nii.pl C<[options]>
+
+Available options are:
+
+-profile  : name of the config file in C<../dicom-archive/.loris_mri>
+
+-minFileID: specifies the minimum FileID to operate on
+
+-maxFileID: specifies the maximum FileID to operate on
+
+-verbose  : be verbose
+
+=head1 DESCRIPTION
+
+This script generates NIfTI images for the inserted MINC files with a FileID
+between C<minFileID> and C<maxFileID> and that are missing NIfTIs.
```

Comment:
just out of curiosity, I quickly looked at the file, it does not seem to check if the NIfTI is created first before calling the make_nii() function, so I am wondering why you say that it only does it for missing NIfTIs?

Reply:
also, if so, you cna re-word the setnece to say something like:
"This script will generate NIfTI images, if non existent, for inserted MINC files ..."

Reply:
oups. Bad wording indeed